packages/react-scripts/template/README.md

```diff
@@ -63,6 +63,7 @@ You can find the most recent version of this guide [here](https://github.com/fac
   - [Now](#now)
   - [S3 and CloudFront](#s3-and-cloudfront)
   - [Surge](#surge)
+- [Advanced Configuration](#advanced-configuration)
```

Comment:
Also let's add this to the main README, like we do for other top level sections.